configure.sh

```diff
@@ -90,57 +95,85 @@ fi
 THREADS_COUNT=${THREADS_COUNT:-1}
 QEMU_PATH=${QEMU_PATH:-"qemu-system-i386"}
 WSL=${WSL:-0}
+WSL_2=${WSL_2:-0}
 SKIP_CC=${SKIP_CC:-0}
 
 # Check for dependencies
-echo "Checking dependencies..."
-DEPS="nasm curl mcopy make"
+echo "[Checking dependencies]"
+DEPS="nasm curl mcopy make mtools"
 for i in $DEPS; do
 	if test -z "$(command -v $i)"; then
 		echo "Error: dependecies not met, $i is not installed."
 		exit 1
 	fi
 done
-echo "DONE"
+echo "[Done]"
 
 # Download files
-echo "Downloading and uncompressing files..."
+echo "[Downloading configuration files]"
 TEMP="/tmp/MicrOS_DevTools_temp"
-SRC="$TEMP/MicrOS-DevTools-2.1"
 mkdir -p "$TEMP"
-curl -Lks https://github.com/jaenek/MicrOS-DevTools/archive/v2.1.tar.gz | tar xzC "$TEMP"
-if test $SKIP_CC -eq 0;then
+curl -Lks "https://raw.githubusercontent.com/JakubPrzystasz/MicrOS-DevTools/wsl2update/build.sh" >> "$TEMP/build.sh" 
+curl -Lks "https://raw.githubusercontent.com/JakubPrzystasz/MicrOS-DevTools/wsl2update/tasks.json" >> "$TEMP/tasks.json" 
+curl -Lks "https://raw.githubusercontent.com/JakubPrzystasz/MicrOS-DevTools/wsl2update/launch.json" >> "$TEMP/launch.json" 
+echo "[Done]"
+if [ $SKIP_CC -eq 0 ]; then
+	echo "[Downloading cross compiler]"
+	echo "!!!Warning!!! This operation can take a while, do not close this window"
 	curl -Lks https://github.com/jaenek/MicrOS-DevTools/releases/download/v1.0/cross.tar.gz | sudo tar xzC "/opt"
 	if test $? -eq 1; then
 		echo "Error: wrong sudo password."
 		exit 1
 	fi
+	echo "[Done]"
 fi
-echo "DONE"
 
 # Replace strings
-echo "Replacing strings..."
-sed -i "s!\[THREADS_COUNT\]!$THREADS_COUNT!g" "$SRC/build.sh"
+echo "[Replacing strings]"
+sed -i "s!\[THREADS_COUNT\]!$THREADS_COUNT!g" "$TEMP/build.sh"
```

Comment:
Add HDD_SIZE replacement
```suggestion
sed -i "s!\[HDD_SIZE\]!$HDD_SIZE!g" "$TEMP/build.sh"
```